pkg/config/setup/config.go

```diff
@@ -120,6 +120,9 @@ const (
 	// DefaultLogsSenderBackoffRecoveryInterval is the default logs sender backoff recovery interval
 	DefaultLogsSenderBackoffRecoveryInterval = 2
 
+	// DefaultHTTPConnectivityRetryIntervalMax is the default maximum interval for HTTP connectivity retry attempts in seconds
+	DefaultHTTPConnectivityRetryIntervalMax = 3600
+
```

Comment:
Any reason to have a const for this ?

We usually try to have the rest of the code not depend on the config for default/constant. The idea being that the value from the config is always the correct one (either the default or the one set by the user).